libs/core-wrappers/src/lib/Navbar/Navbar.module.css

```diff
@@ -7,32 +7,39 @@
 }
 
 .nav-link {
+  text-decoration: none !important;
   min-width: var(--min-width);
   border: none;
-  padding: 0.5rem 1rem;
   padding-left: var(--horizontal-buffer);
   color: #707070;
   cursor: pointer;
   display: block;
 }
-.nav-link:hover {
-  color: #707070;
+
+.nav-content {
+  height: 34px;
   text-decoration: none;
+  padding-left: 2rem;
 }
 
-.nav-link.active {
-  /* TODO: Use a variable here */
-  background: rgba(#9d85ef, 0.25);
+.nav-content:hover {
+  background: var(--global-color-accent--weak);
+}
```

Comment:
**Comment (No Task).** It seems odd to me style `<div>`'s hover, not `<a>`'s. When this gets chiseled, I'll revisit.